client/src/actions.js

```diff
@@ -0,0 +1,32 @@
+let socket
+let _initialized = false
+
+function initialized() {
+    return _initialized
```

Comment:
Yeah definitely a temporary solution.  It's just so that we don't try to start the UI before we know how big the grid is going to be.  The server sends a 'start' message with the size of the grid, then Actions.initialize executes a callback to setup the UI, listen for updates, and run the rendering loop.